app/views/hub/forced_password_resets/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="slab">
+  <div class="grid__item width-one-half shift-one-fourth">
+    <div class="form-card form-card--compact">
+      <div class="form-card__header">
+        <h1 class="form-card__title">Please update your password. </h1>
```

Comment:
Non-strong suggestion: Mark this for translation with `I18n.t`. Use Google Translate to get a translation. Optionally use `GOOGLE_TRANSLATE_API_KEY=xyz i18n-tasks translate-missing` to automatically get the translation so you have an easy time doing it.

(the API key is in LastPass)

In the client/intake-facing code, it's very important to mark strings for translation. In the Hub, it's less important. You'll find a handful of Hub pages that don't use translation, and it's OK not to, and this could be a page where we don't bother. However, since we are so consistent in using translations in the client-facing flow, it's surprising as a reviewer to see inline copy like this. Since this is a weak suggestion from us, it's fine if you decide to leave this comment unaddressed. I'm under the impression there a tiny number of partners who are maybe in Puerto Rico who prefer Spanish, but it's not really a use case we've optimized for.

Also minor: Consider removing the unnecessary ` ` before the `</h1>`.